src/runner.js

```diff
@@ -17,6 +20,13 @@ const Runner = (config: Config, registry: Reg, logger: Object) => {
     for (const m of messages) { // eslint-disable-line
       let params = null;
       try {
+        if ( maxRss ) {
+          const currentRss = memoryUsage().rss;
+          if (currentRss > maxRss) {
+            logger.error(`Steveo - Memory ${currentRss} is above max ${maxRss}. Killing the process`);
```

Comment:
Nah, I don't think the PID really matters to us. It'd be the container/task ID we'd really be interested in anyway. Would be super nice to add those fields to the logs that end up in Loggly as an aside.